src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/InterpolationModel.h

```diff
@@ -94,6 +94,8 @@ namespace OpenMS
       interpolation_ = source.interpolation_;
       scaling_ = source.scaling_;
 
+      updateMembers_();
```

Comment:
why is this needed?

Reply:
It says in the description of updateMembers that it should be at every derived copy constructor/assignment

Reply:
ok makes sense. One thing to watch out as it is easy to just put a = default at a copy constructor/assignment

Reply:
I am actually not sure why updateMembers is needed. Only makes a difference if the params of the source do not match its own members right. But this prevents = default everywhere.

Reply:
I think we would need to check if someone sets the param class in updateMembers (which I think he should not do).